torchmetrics/classification/hamming_distance.py

```diff
@@ -16,7 +16,7 @@
 import torch
 from torch import Tensor, tensor
 
-from torchmetrics.functional.classification.hamming_distance import _hamming_distance_compute, _hamming_distance_update
+from torchmetrics.functional.classification.hamming import _hamming_distance_compute, _hamming_distance_update
```

Comment:
quick ques: just changing the functional metrics files? for eg. this modular file is still `hamming_distance`

Reply:
I see it renamed...